src/jdk.jshell/share/classes/jdk/jshell/execution/LocalExecutionControl.java

```diff
@@ -90,11 +94,19 @@ public void load(ClassBytecodes[] cbcs)
     private static byte[] instrument(byte[] classFile) {
         var cc = ClassFile.of();
         return cc.transformClass(cc.parse(classFile),
-                        ClassTransform.transformingMethodBodies((cob, coe) -> {
-                            if (coe instanceof BranchInstruction)
-                                cob.invokestatic(CD_Cancel, "stopCheck", ConstantDescs.MTD_void);
-                            cob.with(coe);
-                        }));
+                        ClassTransform.transformingMethodBodies(
+                            CodeTransform.ofStateful(() -> {
+                                HashSet<Label> priorLabels = new HashSet<>();
+                                return (builder, element) -> {
+                                    switch (element) {
+                                    case LabelTarget target -> priorLabels.add(target.label());
```

Comment:
I think we usually indentat for `case`/`default`, right? (The old way of putting `case`/`default` at the same indentation level as was the switch is not really used in new code, or is it?)

Reply:
In `jdk.compiler` both variants and appear and there does not appear to be a clear winner. In `jdk.jshell` it appears that indenting is the standard approach.

So we should try to be consistent with the local environment and indent. Thanks - fixed in 16a39b28197.